virttest/libvirt_xml/network_xml.py

```diff
@@ -414,6 +422,50 @@ def __init__(self, virsh_instance=base.virsh):
                                tag_name='virtualport', attribute='type')
         super(NetworkXMLBase, self).__init__(virsh_instance=virsh_instance)
 
+    class Address(base.LibvirtXMLBase):
```

Comment:
Can we use the existing address class in device?

Reply:
The existing address class does not define any attributes, so it will be more complicated to use it to get and set values, or wants to get the value of some attribute.

Reply:
@kylazhang  have updated to use the existing address class in device. Could you help to review it again please? Thanks.